packages/core/src/App/Components/Layout/Header/toggle-menu-drawer.jsx

```diff
@@ -141,6 +142,14 @@ const ToggleMenuDrawer = observer(({ platform_config }) => {
         expandSubMenu(false);
     }, [expandSubMenu, is_open, is_mobile_language_menu_open, setMobileLanguageMenuOpen]);
 
+    const handleLogout = React.useCallback(async () => {
+        toggleDrawer();
+        history.push(routes.traders_hub);
+        await logoutClient();
```

Comment:
[Q] why are we toggling first then logout, don't we want to logout first then toggle?

Reply:
I believe the existing logic was to redirect to traders hub first to act as a logged out state, then logout. I think why this is because some other pages dont have a logged out state so might redirect to oauth. So redirect first then logout